src/components/reusable-ui/Card.jsx

```diff
@@ -1,12 +1,24 @@
-import styled from "styled-components"
-import { theme } from "../../theme"
-import PrimaryButton from "./PrimaryButton"
+import styled from "styled-components";
+import { theme } from "../../theme";
+import PrimaryButton from "./PrimaryButton";
+import { TiDelete } from "react-icons/ti";
+import UserContext from "../../context/UserContext";
+import { useContext, useState } from "react";
+
+const DEFAULT_IMAGE = "/images/coming-soon.png";
+
+export default function Card({ title, imageSource, leftDescription, onClick }) {
+  const { isAdmin } = useContext(UserContext);
 
-export default function Card({ title, imageSource, leftDescription }) {
   return (
-    <CardStyled className="produit">
+    <CardStyled className={isAdmin && "admin-mode"}>
+      {isAdmin && (
+        <button className="delete-icon" onClick={() => onClick(title)}>
+          <TiDelete className="icon" />
+        </button>
+      )}
       <div className="image">
-        <img src={imageSource} alt={title} />
+        <img src={imageSource ? imageSource : DEFAULT_IMAGE} alt={title} />
```

Comment:
```suggestion
        <img src={imageSource} alt={title} />
```

`imageSource ? imageSource : DEFAULT_IMAGE` à déplacer dans le composant Parent au niveau des props de Card.